docs/patterns/components/Table/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Also, with the demo, after changing the sort direction with the keyboard, focus gets lost. I believe it should stay on the sort button

Reply:
I will go with your suggestion unless others have a different opinion. @schne324 @scurker